cloudbridge/cloud/providers/gce/provider.py

```diff
@@ -270,3 +301,19 @@ def wait_for_operation(self, operation, region=None, zone=None):
     def parse_url(self, url):
         out = self._compute_resources.parse_url(url)
         return out if out else self._storage_resources.parse_url(url)
+
+    def get_resource(self, resource, url_or_name):
+        resource_url = (
+            self._compute_resources.get_resource_url_with_default(
+                resource, url_or_name) or
+            self._storage_resources.get_resource_url_with_default(
+                resource, url_or_name))
+        if resource_url is None:
+            return None
+        try:
+            res = resource_url.get_resource()
```

Comment:
nit: could return here and save one line.

Reply:
Done.